### DISCUSSION TITLE
Multiple LDAP sources Docker

### DISCUSSION THREAD

**Discussion by WaaZaa666** [netbox-community/netbox#5457] [Q&A] [answered]
I've been working on my first netbox implementation and have been able to overcome some starting issues. 

Now I'm unable to figure out how to select multiple LDAP sources for netbox. A single source is working just fine, but I would like to add another source. Is there some docs/discussions I did not find?

Running **v2.9.8**, using the docker container. 

relevant config:

```
AUTH_LDAP_SERVER_URI=ldaps://domain.tld
AUTH_LDAP_BIND_DN=CN=netbox_sa,OU=Service Accounts,DC=domain,DC=tld
AUTH_LDAP_BIND_PASSWORD=difficultpassword
AUTH_LDAP_USER_SEARCH_BASEDN=OU=Users,DC=domain,DC=tld
AUTH_LDAP_GROUP_SEARCH_BASEDN=OU=Groups,DC=domain,DC=tld
AUTH_LDAP_REQUIRE_GROUP_DN=CN=.s.netbox-users,OU=Groups,DC=domain,DC=tld
AUTH_LDAP_IS_ADMIN_DN=CN=.s.netbox-admins,OU=Groups,DC=domain,DC=tld
AUTH_LDAP_IS_SUPERUSER_DN=CN=.s.netbox-users,OU=Groups,DC=domain,DC=tld
LDAP_IGNORE_CERT_ERRORS=true
REMOTE_AUTH_ENABLED= True
REMOTE_AUTH_BACKEND='netbox.authentication.LDAPBackend'
```

**Comment by ryanmerolle** [comment, marked as answer]
This leverages the upstream django-auth-ldap project.  I presume if at all possible you would have to leverage the below guide.

https://django-auth-ldap.readthedocs.io/en/latest/multiconfig.html
